vignettes/polars.Rmd

```diff
@@ -390,3 +441,21 @@ pl$DataFrame(iris)$select(
 )$as_data_frame()
 ```
 
+## Data types
+
+Polars is
+[strongly typed](https://en.wikipedia.org/wiki/Strong_and_weak_typing) and new
+types can be created with the `dtypes` constructor. For example:
+
+```{r}
+pl$dtypes$Float64
+```
+
+The full list of valid Polars types can be found by typing `pl$dtypes` 
+into your R console. These include _Boolean_, _Float32(64)_, _Int32(64)_,
+_Utf8_, _Categorical_, _Date_, etc. Note that some type names differ from what
+they are called in R (e.g., _Boolean_ in Polars is equivalent to `logical()` in
+R). This might occasionally require you to look up a specific type. But the good
+news is that **polars** generally does a good job of inferring types
+automatically.
+
```

Comment:
On balance, I agree that moving types to the end makes sense for an intro vignette. But we've already included a type coercion above via `cast`. So it might be an idea to either flag it above, or at least mention something about type coercion here again.